libs/hdf-converters/test/mappers/forward/xccdf_mapper.spec.ts

```diff
@@ -54,6 +54,30 @@ describe('xccdf_mapper', () => {
     });
   });
   describe('OpenSCAP', () => {
+    it('ubuntu1804', () => {
+      const mapper = new XCCDFResultsMapper(
+        fs.readFileSync(
+          'sample_jsons/xccdf_results_mapper/sample_input_report/xccdf-results-openscap-ComplianceAsCode-ubuntu1804.xml',
+          {encoding: 'utf-8'}
+        )
+      );
+
+      // fs.writeFileSync(
```

Comment:
ignore the sonarqube codesmell warning for the commented out code here.  if it has any other findings though, please do fix them.

Reply:
This is the only finding reported by sonarqube; I'll continue to ignore it :)